WcaOnRails/db/migrate/20161117085757_cleanup_events_issue96.rb

```diff
@@ -0,0 +1,12 @@
+class CleanupEventsIssue96 < ActiveRecord::Migration
+  def up
+    remove_foreign_key :preferred_formats, column: :event_id
+    remove_foreign_key :competition_events, column: :event_id
```

Comment:
@pingskills, I suggested the same thing to @jfly. I think we should only delete the events we don't care about. I don't remember why the choice was to delete all in the first place.

@jfly, can you put that reason in here in case anybody else has this same question?